modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/extras/ScaledFloatFieldMapperTests.java

```diff
@@ -196,7 +197,7 @@ public void testStore() throws Exception {
         assertEquals(1230, storedField.numericValue().longValue());
     }
 
-    public void testCoerce() throws Exception {
+    public void testCoerce() throws IOException {
```

Comment:
Changed to make the method signature match how it is declared in `NumberFieldMapperTests`